src/components/scenes/TransactionListScene.js

```diff
@@ -217,14 +218,10 @@ export class TransactionList extends React.Component<Props, State> {
                           <T numberOfLines={1} style={[styles.currentBalanceBoxBits, styles.symbol]}>
                             {displayDenomination.symbol + ' '}
                           </T>
-                          <T numberOfLines={1} style={[styles.currentBalanceBoxBits, styles.symbol]}>
-                            {cryptoAmountString}
-                          </T>
+                          <ThemedTicker style={[styles.currentBalanceBoxBits, styles.symbol]}>{cryptoAmountString}</ThemedTicker>
```

Comment:
ThemedTicker does not honor the "numberOfLines" prop. 
Could that cause any issues down the road? Look into this

Reply:
should be fine. I don’t imagine using spinning numbers on multiple lines